src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,472 @@
+		System.out.println("Please enter the username::");
+		String username = ValidationUtils.emptyValidation(scanner, "Username");
+		System.out.println("Please enter the password::");
+		String password = ValidationUtils.emptyValidation(scanner, "Password");
+		System.out.println("Please enter the full name::");
+		String fullname = ValidationUtils.emptyValidation(scanner, "Full Name");
+		System.out.println("Please enter the department::");
+		String department =
+			ValidationUtils.emptyValidation(scanner, "Department");
+		System.out.println("Please enter the address::");
+		String address = ValidationUtils.emptyValidation(scanner, "Address");
+		System.out.println("Please choose the following role::\n");
+		String role = chooseRole(scanner);
+		employeeServiceImpl = new EmployeeServiceImpl();
+		Employee emp =
+			employeeServiceImpl.saveEmployee(
+				username, password, fullname, department, address, role);
+		if (emp == null) {
+			System.out.println("Username already exists. Please try again.\n");
+		}
+		else if (emp.getId() > 0) {
+			System.out.println(fullname + " is added successfully.\n");
+		}
+
+	}
+
+	/**
+	 * This method is used to search employee using user input
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void searchEmployee(Scanner scanner) {
+
+		List<Employee> empList = new ArrayList<Employee>();
+		System.out.println("Please enter the key::\n");
+		String key = ValidationUtils.keyValidation(scanner);
+		employeeServiceImpl = new EmployeeServiceImpl();
+		empList = employeeServiceImpl.findAll(key);
+		System.out.println("************ Employee List ************\n");
+		String leftAlignFormat = "| %-19s | %-19s | %-21s | %-19s | %-15s |%n";
+		System.out.format("+---------------------+---------------------+-----------------------+---------------------+-----------------+%n");
+		System.out.printf("| Username            | Fullname            | Department            | Address             | Role            |%n");
+		System.out.format("+---------------------+---------------------+-----------------------+---------------------+-----------------+%n");
+		for (Employee emp : empList) {
+			System.out.format(
+				leftAlignFormat, emp.getUsername(), emp.getFullname(),
+				emp.getDepartment(), emp.getAddress(), emp.getRole());
+		}
+		System.out.format("+---------------------+---------------------+-----------------------+-----------------------+----------------+%n");
+		System.out.println("\n");
+	}
+
+
+	/**
+	 * This method is used to delete employee using user input
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void deleteUser(Scanner scanner) {
+
+		System.out.println("Do you want to delete User? Please enter the fullname::\n");
+		String fullname = ValidationUtils.emptyValidation(scanner, "Fullname");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Fullname" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)